### PR TITLE
Cohort page website

### PR DESCRIPTION
**Pull request by lubnaabd**
relate to #56 #58

### CODE REVIEW

File: src/controllers/error.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ exports.client = (req, res) => {
 };
 
 exports.server = (err, req, res, next) => {
+  console.log(err);
```
Comment (ali-7) [inline src/controllers/error.js]: remove console.log

**Review by ali-7 — CHANGES_REQUESTED** [review]

**Review by NouraldinS — APPROVED** [review]

File: src/views/cohortPageWebsite.hbs [hunk 2]
```diff
@@ -61,30 +53,20 @@
...
+          <a href="{{githublink}}">
+            <img src="{{img_url}}">
+          </a>
+          <h4>{{name}}</h4>
+        </div>
+      {{/each}} 
+    {{/if}}
   </section>
+</section>
 </section>
```
Comment (ali-7) [inline src/views/cohortPageWebsite.hbs]: fix indidation

File: src/views/cohortWebsite.hbs [hunk 3]
```diff
@@ -7,7 +7,7 @@
   <section class="cohortList">
     {{#each res}}
```
Comment (ali-7) [inline src/views/cohortWebsite.hbs]: fix indentation

File: src/views/homeWebsite.hbs [hunk 4]
```diff
@@ -1,21 +1,23 @@
...
-    <div class=" w3-display-container">
-      <img class="mySlides" src="/img/besart-bootcamp.jpg">
-      <img class="mySlides" src="/img/besart-bootcamp2.jpg">
-      <img class="mySlides" src="/img/besart-bootcamp3.jpg">
-      <div class="w3-center w3-container w3-section w3-large w3-text-white w3-display-bottommiddle2">
-        <div class="w3-left w3-hover-text-khaki" onclick="plusDivs(-1)">&#10094;</div>
-        <div class="w3-right w3-hover-text-khaki" onclick="plusDivs(1)">&#10095;</div>
-      </div>
-      <div class="w3-center w3-container w3-section w3-large w3-text-white w3-display-bottommiddle">
-        <span class="w3-badge demo w3-border w3-transparent w3-hover-white" onclick="currentDiv(1)"></span>
```
Comment (ali-7) [inline src/views/homeWebsite.hbs]: fix indentation

**Review by ali-7 — CHANGES_REQUESTED** [review]

**Review by ali-7 — APPROVED** [review]

**Pull request merged**